README.md

```diff
@@ -3,13 +3,18 @@
 Generate secrets for the Forgerock Platform
 
 ## Features
-* Optionally store secrets in a Cloud Secret Manager, currently supported:
- * Google Secret Manager
- * AWS Secrets Manager
-* Optionally apply the secrets to the Kubernetes API
+
+The secret-agent generates the secrets required by the ForgeRock Platform. These secretes can be stored in-cluster as Kubernetes secrets. It can also store the secrets in a Cloud Secret Manager. Currently providers supported:
+
+* Google Secret Manager
+* AWS Secrets Manager
 
 ## Usage
-* The default `secretsConfig.yaml` generates all the secrets needed for all the products.
-* Adjust the config as needed.
-* Run the generator in Kubernetes using the example [manifests](manifests), `kubectl apply -f manifests`.
 
+### Deploy
+
+To deploy the secret-agent with default certificates, run:
```

Comment:
are default certs an implementation detail? If so - I'd omit from the readme unless it is pertitent

Reply:
yep. I missed that spot when I cleaned up the README.md 
good catch.